Gruntfile.js

```diff
@@ -583,7 +579,6 @@ module.exports = function (grunt) {
           '<%= unpacked %>/jax/output/SVG/fonts/TeX/Main/Regular/BasicLatin.js',
           '<%= unpacked %>/jax/output/SVG/fonts/TeX/Main/Regular/CombDiacritMarks.js',
           '<%= unpacked %>/jax/output/SVG/fonts/TeX/Main/Regular/GeometricShapes.js',
-          '<%= unpacked %>/jax/output/SVG/fonts/TeX/Main/Regular/MathOperators.js',
```

Comment:
This is another file that I found which is not available in the MathJax distribution.